salt/modules/mount.py

```diff
@@ -1422,3 +1569,287 @@ def delete_mount_cache(real_name):
                 if not cache_write:
                     raise CommandExecutionError('Unable to write mount cache.')
     return True
+
+
+def _filesystems(config='/etc/filesystems', leading_key=True):
+    '''
+    .. versionadded:: 2018.3.3
```

Comment:
Sphinx directives aren't really useful in a private function.

Reply:
done